### ISSUE TITLE
GET api/request filter "onlyMine" does not seem to work

### ISSUE THREAD

**Issue by andre-stefanov** [NexdApp/nexd-backend#34]
Trying to reduce retrieved requests to just the ones created by a user does not work, response still contains requests from other users.

```
GET /api/request?onlyMine=true HTTP/1.1
Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJmaXJzdE5hbWUiOiJBbmRyZSIsImxhc3ROYW1lIjoiUG9zdG1hbiIsImVtYWlsIjoicG9zdG1hbkBhbmRyZS5kZSIsInN0cmVldCI6bnVsbCwibnVtYmVyIjpudWxsLCJ6aXBDb2RlIjpudWxsLCJjaXR5IjpudWxsLCJ0ZWxlcGhvbmUiOm51bGwsImlkIjo2LCJyb2xlIjoibm9uZSIsImlhdCI6MTU4NDkyNjk4NCwiZXhwIjoxNTg0OTMwNTg0fQ.h4GytYoAGZ64rKCUuorB7tofwY-pqYwr05gJby7zf0g
User-Agent: PostmanRuntime/7.23.0
Accept: */*
Cache-Control: no-cache
Postman-Token: cf66ed23-2491-4b80-bfe6-e5d17634f9a9
Host: nexd-api-alb-1107636132.eu-central-1.elb.amazonaws.com
Accept-Encoding: gzip, deflate, br
Connection: keep-alive
HTTP/1.1 200 OK
Date: Mon, 23 Mar 2020 01:59:20 GMT
Content-Type: application/json; charset=utf-8
Content-Length: 5827
Connection: keep-alive
X-DNS-Prefetch-Control: off
X-Frame-Options: SAMEORIGIN
Strict-Transport-Security: max-age=15552000; includeSubDomains
X-Download-Options: noopen
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Access-Control-Allow-Origin: *
ETag: W/"16c3-baMOPgy4RGbydQTuymd0LEXfh24"
[{"street":null,"number":null,"zipCode":"12345","city":"München","id":1,"created_at":"2020-03-22T21:22:56.205Z","requesterId":1,"priority":"low","additionalRequest":"","deliveryComment":"","phoneNumber":"","status":"new","articles":[{"id":1,"articleId":1,"articleCount":1,"articleDone":false},{"id":2,"articleId":3,"articleCount":1,"articleDone":false}]},{"street":null,"number":null,"zipCode":null,"city":null,"id":2,"created_at":"2020-03-22T22:17:46.153Z","requesterId":3,"priority":"low","additionalRequest":"","deliveryComment":"","phoneNumber":"","status":"new","articles":[{"id":3,"articleId":2,"articleCount":1,"articleDone":false},{"id":4,"articleId":3,"articleCount":1,"articleDone":false}]},{"street":null,"number":null,"zipCode":null,"city":null,"id":3,"created_at":"2020-03-22T22:18:00.210Z","requesterId":3,"priority":"low","additionalRequest":"","deliveryComment":"","phoneNumber":"","status":"new","articles":[{"id":5,"articleId":2,"articleCount":1,"articleDone":false},{"id":6,"articleId":3,"articleCount":1,"articleDone":false},{"id":7,"articleId":5,"articleCount":1,"articleDone":false}]},{"street":null,"number":null,"zipCode":null,"city":null,"id":4,"created_at":"2020-03-22T22:18:45.366Z","requesterId":3,"priority":"low","additionalRequest":"","deliveryComment":"","phoneNumber":"","status":"new","articles":[{"id":8,"articleId":2,"articleCount":1,"articleDone":false},{"id":9,"articleId":3,"articleCount":1,"articleDone":false},{"id":10,"articleId":5,"articleCount":1,"articleDone":false}]},{"street":null,"number":null,"zipCode":null,"city":null,"id":5,"created_at":"2020-03-22T23:20:20.733Z","requesterId":5,"priority":"low","additionalRequest":"bla","deliveryComment":null,"phoneNumber":null,"status":"new","articles":[{"id":11,"articleId":1,"articleCount":1,"articleDone":false},{"id":12,"articleId":2,"articleCount":1,"articleDone":false},{"id":13,"articleId":3,"articleCount":0,"articleDone":false},{"id":14,"articleId":4,"articleCount":1,"articleDone":false},{"id":15,"articleId":5,"articleCount":0,"articleDone":false},{"id":16,"articleId":6,"articleCount":1,"articleDone":false},{"id":17,"articleId":7,"articleCount":0,"articleDone":false}]},{"street":null,"number":null,"zipCode":null,"city":null,"id":6,"created_at":"2020-03-23T01:13:07.434Z","requesterId":5,"priority":"low","additionalRequest":"","deliveryComment":null,"phoneNumber":null,"status":"new","articles":[{"id":18,"articleId":1,"articleCount":2,"articleDone":false},{"id":19,"articleId":2,"articleCount":0,"articleDone":false},{"id":20,"articleId":3,"articleCount":2,"articleDone":false},{"id":21,"articleId":4,"articleCount":0,"articleDone":false},{"id":22,"articleId":5,"articleCount":1,"articleDone":false},{"id":23,"articleId":6,"articleCount":0,"articleDone":false},{"id":24,"articleId":7,"articleCount":1,"articleDone":false}]},{"street":null,"number":null,"zipCode":null,"city":null,"id":7,"created_at":"2020-03-23T01:18:18.030Z","requesterId":5,"priority":"low","additionalRequest":"","deliveryComment":null,"phoneNumber":null,"status":"new","articles":[{"id":25,"articleId":1,"articleCount":0,"articleDone":false},{"id":26,"articleId":2,"articleCount":0,"articleDone":false},{"id":27,"articleId":3,"articleCount":0,"articleDone":false},{"id":28,"articleId":4,"articleCount":0,"articleDone":false},{"id":29,"articleId":5,"articleCount":0,"articleDone":false},{"id":30,"articleId":6,"articleCount":0,"articleDone":false},{"id":31,"articleId":7,"articleCount":0,"articleDone":false}]},{"street":null,"number":null,"zipCode":null,"city":null,"id":8,"created_at":"2020-03-23T01:43:19.976Z","requesterId":5,"priority":"low","additionalRequest":"","deliveryComment":null,"phoneNumber":null,"status":"new","articles":[{"id":32,"articleId":1,"articleCount":1,"articleDone":false},{"id":33,"articleId":2,"articleCount":1,"articleDone":false},{"id":34,"articleId":3,"articleCount":1,"articleDone":false},{"id":35,"articleId":4,"articleCount":0,"articleDone":false},{"id":36,"articleId":5,"articleCount":0,"articleDone":false},{"id":37,"articleId":6,"articleCount":0,"articleDone":false},{"id":38,"articleId":7,"articleCount":0,"articleDone":false},{"id":39,"articleId":8,"articleCount":0,"articleDone":false},{"id":40,"articleId":9,"articleCount":0,"articleDone":false}]},{"street":null,"number":null,"zipCode":null,"city":null,"id":9,"created_at":"2020-03-23T01:45:56.649Z","requesterId":5,"priority":"low","additionalRequest":"","deliveryComment":null,"phoneNumber":null,"status":"new","articles":[{"id":41,"articleId":1,"articleCount":0,"articleDone":false},{"id":42,"articleId":2,"articleCount":0,"articleDone":false},{"id":43,"articleId":3,"articleCount":1,"articleDone":false},{"id":44,"articleId":4,"articleCount":1,"articleDone":false},{"id":45,"articleId":5,"articleCount":0,"articleDone":false},{"id":46,"articleId":6,"articleCount":1,"articleDone":false},{"id":47,"articleId":7,"articleCount":0,"articleDone":false},{"id":48,"articleId":8,"articleCount":1,"articleDone":false},{"id":49,"articleId":9,"articleCount":0,"articleDone":false}]},{"street":null,"number":null,"zipCode":null,"city":null,"id":10,"created_at":"2020-03-23T01:54:54.200Z","requesterId":6,"priority":"low","additionalRequest":"","deliveryComment":null,"phoneNumber":null,"status":"new","articles":[{"id":50,"articleId":1,"articleCount":0,"articleDone":false},{"id":51,"articleId":2,"articleCount":0,"articleDone":false},{"id":52,"articleId":3,"articleCount":1,"articleDone":false},{"id":53,"articleId":4,"articleCount":1,"articleDone":false},{"id":54,"articleId":5,"articleCount":0,"articleDone":false},{"id":55,"articleId":6,"articleCount":1,"articleDone":false},{"id":56,"articleId":7,"articleCount":0,"articleDone":false},{"id":57,"articleId":8,"articleCount":1,"articleDone":false},{"id":58,"articleId":9,"articleCount":0,"articleDone":false}]}]
```

**Comment by fanick-ber** [comment]
fixed in PR #47 , will also be changed (different request) in PR https://github.com/NexdApp/nexd-backend/pull/46

**Comment by fanick-ber** [comment]
This is now resolved, correct?
